python/tank/pipelineconfig.py

```diff
@@ -858,31 +860,31 @@ def get_core_hooks_location(self):
         
         :returns: path string
         """
-        return os.path.join(self._pc_root, "config", "core", "hooks")
+        return os.path.join(self.get_config_location(), "core", "hooks")
 
     def get_schema_config_location(self):
         """
         Returns the location of the folder schema
         
         :returns: path string
         """
-        return os.path.join(self._pc_root, "config", "core", "schema")
+        return os.path.join(self.get_config_location(), "core", "schema")
 
     def get_config_location(self):
         """
         Returns the config folder for the project
         
         :returns: path string
         """
-        return os.path.join(self._pc_root, "config")
+        return self._descriptor.get_config_folder()
 
     def get_hooks_location(self):
         """
         Returns the hooks folder for the project
         
         :returns: path string
         """
-        return os.path.join(self._pc_root, "config", "hooks")
+        return os.path.join(self.get_config_location(), "hooks")
```

Comment:
Hooks are pulled from the configuration (bundle cache or classic config/hooks folder)